settings.gradle.kts

```diff
@@ -1,3 +1,8 @@
+pluginManagement {
+    plugins {
+        kotlin("jvm") version "2.0.21"
+    }
+}
```

Comment:
```suggestion
```

Plugins are managed in `build.gradle.kts`, Versions are manage through `./gradle/libs.versions.toml`. Also the Kotlin 2.0 upgrade requires evaluation to see whether it works with older IntelliJ versions.